lib/plugin_config.js

```diff
@@ -11,7 +11,9 @@ module.exports = function (root) {
   var buildSourcePatterns = [
     'package.json',
     'index.js',
-    '{lib,public,server,webpackShims}/**/*',
+    'npm-shrinkwrap.json',
```

Comment:
When these files don't exist they cause errors to be thrown, maybe there is an argument we can pass to `vfs.src()`?

![image](https://cloud.githubusercontent.com/assets/1329312/21415555/342d171e-c7c8-11e6-856f-d259dce6b446.png)